src/pages/productDetailsPage/ProductDetails.jsx

```diff
@@ -104,6 +106,8 @@ const ProductDetails = () => {
     }
   }, []);
 
+  useEffect(() => {}, [hit]);
```

Comment:
probably we dont need this, as the useEffect is empty its redundant, changes on `hit` should force a re-render even without this.